lib/licence_details_from_licensify.rb

```diff
@@ -1,11 +1,11 @@
-class LicenceDetailsFromArtefact
+class LicenceDetailsFromLicensify
   attr_accessor :licence, :snac_code, :licence_authority_slug, :interaction
 
   def initialize(artefact, licence_authority_slug, snac_code, interaction = nil)
-    self.licence = artefact['details']['licence']
-    self.snac_code = snac_code
-    self.licence_authority_slug = licence_authority_slug
-    self.interaction = interaction
+    @artefact = artefact
```

Comment:
Does this object need an `Artefact`?  Might it be simpler if instantiated with the `licence_identifier` instead?